sos/plugins/__init__.py

```diff
@@ -949,7 +949,8 @@ def file_grep(self, regexp, *fnames):
         return grep(regexp, *fnames)
 
     def _mangle_command(self, exe):
-        name_max = self.archive.name_max()
+        # shorten the name_max such that we can append "_2" in case of conflict
+        name_max = self.archive.name_max() - 5
```

Comment:
Why five?

Reply:
Any value bigger than1 would fit - I just wanted to have enough free space for _10, _100 etc., just in case. But this is redundant change, now.